specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2020-08-06-preview/baremetalinfrastructure.json

```diff
@@ -0,0 +1,890 @@
+            "$ref": "./examples/AzureBareMetalOperations_List.json"
+          }
+        },
+        "description": "Gets a list of AzureBareMetal management operations.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BareMetalInfrastructure/bareMetalInstances/{azureBareMetalInstanceName}": {
+      "get": {
+        "tags": [
+          "BareMetalInfrastructure"
+        ],
+        "operationId": "AzureBareMetalInstances_Get",
+        "summary": "Gets an Azure BareMetal instance.",
+        "description": "Gets an Azure BareMetal instance for the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get an AzureBareMetal instance": {
+            "$ref": "./examples/AzureBareMetalInstances_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureBareMetalInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "BareMetalInfrastructure"
+        ],
+        "operationId": "AzureBareMetalInstances_Delete",
+        "x-ms-long-running-operation": true,
+        "summary": "Deletes a Azure BareMetal instance.",
+        "description": "Deletes a Azure BareMetal instance with the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Delete an AzureBareMetal instance": {
+            "$ref": "./examples/AzureBareMetalInstances_Delete.json"
+          }
+        },
+        "parameters": [
```

Comment:
I would STRONGLY suggest you to remove these duplicated common definitions and rely on those defined under Common-type/v1 or v2's type.json. There are many examples in the [repo.](https://github.com/Azure/azure-rest-api-specs/blob/8fee11f5dd30e228a850ee25a2f3368dc1671b53/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json#L33)

Reply:
done for the common definitions